cxone-chat-sdk/.swiftlint.yml

```diff
@@ -62,3 +79,20 @@ type_name:
     error: 45
   excluded: id
 reporter: "xcode"
+
+file_header:
+  required_pattern: |
+                    \/\/
+                    \/\/ Copyright \(c\) 2021-\d{4}. NICE Ltd. All rights reserved.
+                    \/\/
+                    \/\/ Licensed under the NICE License\;
+                    \/\/ you may not use this file except in compliance with the License.
+                    \/\/ You may obtain a copy of the License at
+                    \/\/
+                    \/\/    https:\/\/github.com\/BrandEmbassy\/cxone-mobile-sdk-IOS\/blob\/master\/LICENSE
```

Comment:
URL update - also here

Reply:
Don't know why GitHub does not present it as changed but if I access the file via file under the branch, it has been changed correctly - https://github.com/nice-devone/nice-cxone-mobile-sdk-ios/blob/feature/License/cxone-chat-sdk/.swiftlint.yml.